ecosystem/sep-0006.md

```diff
@@ -455,7 +476,7 @@ The `fields` object contains a key for each field name and an object with the fo
 
 The wallet should display a form to the user to fill out any fields with unknown values as part of the deposit/withdrawal flow. Each field should be an text `input`, unless `choices` is specified, in which case a dropdown should be used.
 
-An anchor should also indicate in the `/info` response if they support the following optional endpoints:
+An anchor should also indicate in the `/info` response if they support the following optional endpoints and if they require authentication:
```

Comment:
Now that there is more than one field that should be provided for each endpoint, should we list them here to keep consistency? Something in the lines of:

An anchor should also indicate in the `/info` response if they support the `fee`, `transactions` and `transaction` endpoints, by providing the following fields for each:

* `enabled`: `true` if the endpoint is available.
*  `authentication_required`:`true` if client must be [authenticated](#authentication) before accessing the endpoint.